provider/inspec.rb

```diff
@@ -269,6 +271,13 @@ def grab_attributes
       YAML.load_file('templates/inspec/tests/integration/configuration/mm-attributes.yml')
     end
 
+    def old_attribute(attribute_name, doc_generation = false)
```

Comment:
I'm a little confused about what "old_attribute" does. Could you rename / add a comment?

Reply:
Added a comment. I'm not really sure about the best way to name this, it is based off of variables that are outside of MM source control

Reply:
Maybe something like `external_attribute`?